develop/automatic-testing.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Automatic Testing
+description: A guide to writing automatic tests with Fabric Loader JUnit.
+authors:
+  - kevinthegreat1
+---
+
+# Automatic Testing {#automatic-testing}
+
+This page explains how to write code to automatically test parts of your mod.
+There are two ways to automatically test your mod:
+units tests with Fabric Loader JUnit or game tests with the Gametest framework from Minecraft.
+Units tests should be used to test components of your code such as methods and helper classes
+while gametests spins up an entire Minecraft client and server to run your tests
+which makes it more suitable for testing features and gameplay.
+Currently, this guide only covers unit testing.
+
+## Unit Testing with Fabric Loader JUnit {#unit-testing-with-fabric-loader-junit}
+
+Due to the nature of Minecraft modding such as obfuscation, simply adding and using JUnit normally will not work.
```

Comment:
obfuscation doesnt have anything to do with it, it all happens in a development enviroment.

The issues basically come from the need to apply mixins during unit tests. I would change this to something along the lines of:

`Due to the nature of Minecraft modding using runtime bytecode modification tools such as Mixin`